target_chains/starknet/contracts/src/pyth.cairo

```diff
@@ -98,7 +98,9 @@ mod pyth {
     struct Storage {
         wormhole_address: ContractAddress,
         fee_token_address: ContractAddress,
```

Comment:
i think renaming the existing one to `fee_token_address1` (or `_1`) makes it more verbose.